### PR TITLE
NavBar Component

### PR DESCRIPTION
**Pull request by skiran86**

### CODE REVIEW

**Comment by skiran86** [comment]
Pending items - Remove following from navBarView:
import queryString from 'query-string';
import { NavLink } from 'react-router-dom';
import { useSelector } from 'react-redux';
import env from '../../utils/env';

**Comment by skiran86** [comment]
Need to find a way to handle 'initCart'

File: src/components/navBar/NavBarView.js [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+  // Empty second argument of react useEffect will avoid the infinte loop that
+  // caused due to component update
+  const [clickedEl, setClickedEl] = React.useState(null);
+
+  function handleButtonClickEvent(eventName) {
+    setClickedEl(eventName);
+  }
+
+  React.useEffect(() => {
+    const values = {};
```
Comment (AjayDoddapaneni) [inline src/components/navBar/NavBarView.js:36]: Can you clean up or comment it out this as we are no longer using it

Comment (skiran86) [reply src/components/navBar/NavBarView.js]: Done

File: src/components/navBar/NavBarView.js [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+  //         // Ajay Need to update this
+  //         // setAuthState(
+  //         //  { ...authState, username: "", isAuthorized: false }
+  //         //  );
+  //         // localStorage.setItem("isAuthorized", "false");
+  //       });
+  //   }
+  // }, []);
+
+  const numberOfCases = 0;
```
Comment (AjayDoddapaneni) [inline src/components/navBar/NavBarView.js:64]: Can you pass this as a prop and default it to zero?

Comment (skiran86) [reply src/components/navBar/NavBarView.js]: Done

**Review by AjayDoddapaneni — APPROVED** [review]

**Pull request merged**